### ISSUE TITLE
no admin page

### ISSUE THREAD

**Issue by dgiffen** [NNTmux/newznab-tmux#700]
Ubuntu 16.04 server. I got the install finished. NNTmux installation completed successfully. I didn't encounter any errors during the entire install via the guide. I then try to access the admin/group-list, but I am only getting the index.php from the public folder. No firewall is running. I don't have any other programs or web sites via the var/www folder.

**Comment by DariusIII** [comment]
You mean that you are getting the login page?

**Comment by dgiffen** [comment]
I followed the Ubuntu web guide. PHP version is PHP 7.2.9-1. Everything
installed properly. I get to this part
http://IpAddressOfYourServer/admin/group-list
<http://ipaddressofyourserver/admin/group-list> I changed it to my server,
I also tried its IP number. same result. Instead of a admin/group-list page
I get the index.php page from the public folder.

Thanks
Doug

**Comment by DariusIII** [comment]
You mean the contents of the index.php? Have you done `composer install` ?

**Comment by dgiffen** [comment]
Yes. I followed the guide. Everything installed correctly. I did this twice
and both times I get the same result. I never received any errors during
install.
Thanks
Doug

**Comment by DariusIII** [comment]
Can you add a screenshot of what you are seeing?

**Comment by dgiffen** [comment]
I get a different shot from firefox vs chrome. I included both here.
Thanks
Doug

**Comment by DariusIII** [comment]
And what do you see if you don't go to admin? 

Are you using apache or nginx? Have you created config for your site, as per instructions?

**Comment by dgiffen** [comment]
Yes. All PHP extensions listed on the guide were installed. Could an older
php version be breaking this down.

Thanks
Doug

**Comment by DariusIII** [comment]
In no way you should see contents of index.php. If you see it, that means that apache is missing some mods, like rewrite etc.

**Comment by DariusIII** [comment]
Can you paste me contents of your site conf file?

**Comment by dgiffen** [comment]
<VirtualHost *:80>
    ServerAdmin webmaster@localhost
    ServerName localhost
    DocumentRoot "/var/www/NNTmux/public"
    LogLevel warn
    ServerSignature Off
    ErrorLog /var/log/apache2/error.log
    <Directory "/var/www/NNTmux/public">
        Options FollowSymLinks
        AllowOverride All
        Require all granted
    </Directory>
    Alias /covers /var/www/NNTmux/resources/covers
</VirtualHost>
apache2.conf
# This is the main Apache server configuration file.  It contains the
# configuration directives that give the server its instructions.
# See http://httpd.apache.org/docs/2.4/ for detailed information about
# the directives and /usr/share/doc/apache2/README.Debian about Debian
specific
# hints.
#
#
# Summary of how the Apache 2 configuration works in Debian:
# The Apache 2 web server configuration in Debian is quite different to
# upstream's suggested way to configure the web server. This is because
Debian's
# default Apache2 installation attempts to make adding and removing modules,
# virtual hosts, and extra configuration directives as flexible as
possible, in
# order to make automating the changes and administering the server as easy
as
# possible.

# It is split into several files forming the configuration hierarchy
outlined
# below, all located in the /etc/apache2/ directory:
#
# /etc/apache2/
# |-- apache2.conf
# | `--  ports.conf
# |-- mods-enabled
# | |-- *.load
# | `-- *.conf
# |-- conf-enabled
# | `-- *.conf
# `-- sites-enabled
# `-- *.conf
#
#
# * apache2.conf is the main configuration file (this file). It puts the
pieces
#   together by including all remaining configuration files when starting
up the
#   web server.
#
# * ports.conf is always included from the main configuration file. It is
#   supposed to determine listening ports for incoming connections which
can be
#   customized anytime.
#
# * Configuration files in the mods-enabled/, conf-enabled/ and
sites-enabled/
#   directories contain particular configuration snippets which manage
modules,
#   global configuration fragments, or virtual host configurations,
#   respectively.
#
#   They are activated by symlinking available configuration files from
their
#   respective *-available/ counterparts. These should be managed by using
our
#   helpers a2enmod/a2dismod, a2ensite/a2dissite and a2enconf/a2disconf. See
#   their respective man pages for detailed information.
#
# * The binary is called apache2. Due to the use of environment variables,
in
#   the default configuration, apache2 needs to be started/stopped with
#   /etc/init.d/apache2 or apache2ctl. Calling /usr/bin/apache2 directly
will not
#   work with the default configuration.


# Global configuration
#

#
# ServerRoot: The top of the directory tree under which the server's
# configuration, error, and log files are kept.
#
# NOTE!  If you intend to place this on an NFS (or otherwise network)
# mounted filesystem then please read the Mutex documentation (available
# at <URL:http://httpd.apache.org/docs/2.4/mod/core.html#mutex>);
# you will save yourself a lot of trouble.
#
# Do NOT add a slash at the end of the directory path.
#
#ServerRoot "/etc/apache2"

#
# The accept serialization lock file MUST BE STORED ON A LOCAL DISK.
#
Mutex file:${APACHE_LOCK_DIR} default

#
# PidFile: The file in which the server should record its process
# identification number when it starts.
# This needs to be set in /etc/apache2/envvars
#
PidFile ${APACHE_PID_FILE}

#
# Timeout: The number of seconds before receives and sends time out.
#
Timeout 300

#
# KeepAlive: Whether or not to allow persistent connections (more than
# one request per connection). Set to "Off" to deactivate.
#
KeepAlive On

#
# MaxKeepAliveRequests: The maximum number of requests to allow
# during a persistent connection. Set to 0 to allow an unlimited amount.
# We recommend you leave this number high, for maximum performance.
#
MaxKeepAliveRequests 100

#
# KeepAliveTimeout: Number of seconds to wait for the next request from the
# same client on the same connection.
#
KeepAliveTimeout 5


# These need to be set in /etc/apache2/envvars
User ${APACHE_RUN_USER}
Group ${APACHE_RUN_GROUP}

#
# HostnameLookups: Log the names of clients or just their IP addresses
# e.g., www.apache.org (on) or 204.62.129.132 (off).
# The default is off because it'd be overall better for the net if people
# had to knowingly turn this feature on, since enabling it means that
# each client request will result in AT LEAST one lookup request to the
# nameserver.
#
HostnameLookups Off

# ErrorLog: The location of the error log file.
# If you do not specify an ErrorLog directive within a <VirtualHost>
# container, error messages relating to that virtual host will be
# logged here.  If you *do* define an error logfile for a <VirtualHost>
# container, that host's errors will be logged there and not here.
#
ErrorLog ${APACHE_LOG_DIR}/error.log

#
# LogLevel: Control the severity of messages logged to the error_log.
# Available values: trace8, ..., trace1, debug, info, notice, warn,
# error, crit, alert, emerg.
# It is also possible to configure the log level for particular modules,
e.g.
# "LogLevel info ssl:warn"
#
LogLevel warn

# Include module configuration:
IncludeOptional mods-enabled/*.load
IncludeOptional mods-enabled/*.conf

# Include list of ports to listen on
Include ports.conf


# Sets the default security model of the Apache2 HTTPD server. It does
# not allow access to the root filesystem outside of /usr/share and
/var/www.
# The former is used by web applications packaged in Debian,
# the latter may be used for local directories served by the web server. If
# your system is serving content from a sub-directory in /srv you must allow
# access here, or in any related virtual host.
<Directory />
Options FollowSymLinks
AllowOverride None
Require all denied
</Directory>

<Directory /usr/share>
AllowOverride None
Require all granted
</Directory>

<Directory /var/www/>
Options Indexes FollowSymLinks
AllowOverride All
Require all granted
</Directory>

#<Directory /srv/>
# Options Indexes FollowSymLinks
# AllowOverride None
# Require all granted
#</Directory>




# AccessFileName: The name of the file to look for in each directory
# for additional configuration directives.  See also the AllowOverride
# directive.
#
AccessFileName .htaccess

#
# The following lines prevent .htaccess and .htpasswd files from being
# viewed by Web clients.
#
<FilesMatch "^\.ht">
Require all denied
</FilesMatch>


#
# The following directives define some format nicknames for use with
# a CustomLog directive.
#
# These deviate from the Common Log Format definitions in that they use %O
# (the actual bytes sent including headers) instead of %b (the size of the
# requested file), because the latter makes it impossible to detect partial
# requests.
#
# Note that the use of %{X-Forwarded-For}i instead of %h is not recommended.
# Use mod_remoteip instead.
#
LogFormat "%v:%p %h %l %u %t \"%r\" %>s %O \"%{Referer}i\"
\"%{User-Agent}i\"" vhost_combined
LogFormat "%h %l %u %t \"%r\" %>s %O \"%{Referer}i\" \"%{User-Agent}i\""
combined
LogFormat "%h %l %u %t \"%r\" %>s %O" common
LogFormat "%{Referer}i -> %U" referer
LogFormat "%{User-agent}i" agent

# Include of directories ignores editors' and dpkg's backup files,
# see README.Debian for details.

# Include generic snippets of statements
IncludeOptional conf-enabled/*.conf

# Include the virtual host configurations:
IncludeOptional sites-enabled/*.conf

# vim: syntax=apache ts=4 sw=4 sts=4 sr noet
thanks
Doug

**Comment by dgiffen** [comment]
yes, I have rebooted a few times to see if that would clear up any issues.
still the same. Tried from chrome on a mint destop. I also tried with an
ipad running safari. Same page every time.
Thanks
doug

**Comment by DariusIII** [comment]
Edit your .env file and set APP_ENV to local and DEBUG to true. Open your site and see if anything goes through.

**Comment by dgiffen** [comment]
Firefox
make(\Illuminate\Contracts\Http\Kernel::class); $response =
$kernel->handle($request = \Illuminate\Http\Request::capture());
$response->send(); $kernel->terminate($request, $response);
Chrome
<?php

require_once dirname(__DIR__).DIRECTORY_SEPARATOR.'resources/views/themes/smarty.php';

$kernel = $app->make(\Illuminate\Contracts\Http\Kernel::class);

$response = $kernel->handle($request = \Illuminate\Http\Request::capture());
$response->send();

$kernel->terminate($request, $response);

Not sure why they are different. Both are newest versions in the Mint
repos. Both server and desktop have been fully updated.

Thanks
Doug

**Comment by dgiffen** [comment]
I will later today and get back with the results

Thanks
Doug

**Comment by dgiffen** [comment]
I tried the fix. Same result. The httpd.conf seems to be merged into
apache.conf

<FilesMatch ".+\.ph(p[345]?|t|tml)$">
    SetHandler application/x-httpd-php

</FilesMatch>
I placed it in the apache.conf. I also found the same block in the

Thanks
Doug

**Comment by DariusIII** [comment]
I really don't know how to help you, as the issue is not NNTmux but Apache misconfiguration related.

**Comment by bert0901** [comment]
Not sure if this helps but I had similar issue.

in the .env file make sure the following are the same domain address :

APP_URL=
SESSION_DOMAIN=

**Comment by dgiffen** [comment]
Thanks for the help here. I already decided to Nuke the entire installation
before I saw your text. Now I am having issues getting the php-imagick to
install. I am going to move on. I was trying to install this on a server
that has been in use for a few years. I may give it another go in a few
months. I will try again on a fresh install.
Thanks
Doug

**Comment by dgiffen** [comment]
I understand. I didn't get a chance to try another suggestion. I didn't see
the suggestion until I had already Nuked the entire install. I removed PHP,
Maria, Web Server. I then tried to start from step one and have encountered
errors getting php-imagick to install. I'm going to move on here. I may
give this a go in a few months on a clean install. Your program looked
really good and I appreciate your help.
Thanks
Doug


On Sep 5, 2018 7:41 AM, "DariusIII" <notifications@github.com> wrote:

I really don't know how to help you, as the issue is not NNTmux but Apache
misconfiguration related.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/NNTmux/newznab-tmux/issues/700#issuecomment-418700198>,
or mute the thread
